src/components/ShoppingList/ShoppingList.js

```diff
@@ -40,17 +47,44 @@ function ShoppingList({ listId }) {
       );
     } else {
       return (
-        <ul className="shopping-list__list list-reset">
-          {listItems.docs.map((doc) => (
-            <ShoppingListItem
-              key={doc.id}
-              listId={listId}
-              itemId={doc.id}
-              item={doc.data()}
-              handleCheck={handleCheck}
+        <div>
+          <div className="filter">
+            <label htmlFor="filterInput" className="filter__label label">
+              Filter items
+            </label>
+            <input
+              type="text"
+              id="filterInput"
+              name="filterInput"
+              value={filter}
+              onChange={handleInput}
+              className="filter__text-field text-field"
             />
-          ))}
-        </ul>
+            <button
+              type="button"
+              aria-label="clear"
```

Comment:
Do we want the`aria-label` to also read "clear filter" since the text within the button is also "Clear Filter"? When I turn on a screenreader, it reads "clear, button" whereas if we add "clear filter" for the `aria-label` it reads "clear filter button". I think it'd be more clear.

Reply:
Yes, I agree, or even "Clear item filter". 

Also want to note, because I may not have communicated this clearly earlier: We don't currently need an aria-label *and* text inside the button, one or the other is fine. Since we discussed down the line removing the text inside the button tags and making it an image instead, and in that case *would* need the aria-label, it doesn't hurt to keep it. But I just want to make sure we're clear on when an aria-label is necessary and when it's not—screen readers will read inner text of a button, and an aria-label will override that text if it's provided.

Reply:
That makes sense, and yes I think I just left the tex in there for now for the purposes of getting through this review cycle, and then we can remove it later once we add that styling.